tiptapy/__init__.py

```diff
@@ -69,6 +69,14 @@ def inner_render(self, node) -> str:
         return html
 
 
+class Embed(BaseContainer):
+    type = "embed"
+
+    def inner_render(self, node):
+        attrs = node['attrs']
+        return attrs.get('html', '')
```

Comment:
@shon now it just returns an empty string when nothin is found instead of `None`.